learn/advanced/known_limitations.md

```diff
@@ -82,3 +82,9 @@ user = 1 OR user = 2 […] OR user = 1500 OR user = 1501 […] OR user = 2000 OR
   ["user = 1", "user = 2", […], "user = 1500", "user = 1501", […], "user = 2000", "user = 2001"]
 ]
 ```  
+
+## Size of numeric fields
+
+**Limitation:** Meilisearch can only exactly represent integers between -2⁵³ and 2⁵³.
```

Comment:
I'm worried about the use of the word integer here. Since I learned to program with C, integer to me means 32-bit integer, so between -2147483648 and 2147483647. That's a much smaller range (-2<sup>15</sup> to 2<sup>15</sup>). I'm probably not alone in this.

```suggestion
**Limitation:** Meilisearch can only exactly represent numeric values between -2⁵³ and 2⁵³.
```

Reply:
Maybe we should rename this to `Size of integer fields`. From what I understand, it can represent decimal values, but this double-precision floating-point format allows you to exactly represent integers between -2⁵³ and 2⁵³.
Does that make sense?

Reply:
The mathematical + computing definition of integer AFAIK does not include decimals (AKA the infinite range of values between two whole numbers). 

Meilisearch does use the double-precision floating-point type under the hood, but in terms of what's being exposed to our users, it's just a numeric field with no specific type (and can include decimals, so using the term "integer" would be actively misleading here).

Reply:
Ok, quite honestly, a lot of the subtleties of this discussion go over my head because floating-point operations are a Weird™ subject. All I can say for sure is that the linked issue explicitly names integers and I don't think a core dev would use that term lightly.

But let's take the guesswork out of this. @maryamsulemani97, please address and resolve all remaining open conversations in this PR. Then, request kéro's review. If you want to get to the bottom of this, you might want to get explicit confirmation the limitation we are documenting here concerns integers only.